src/router/index.js

```diff
@@ -85,4 +85,7 @@ export default new Router({
       },
     },
   ],
+  scrollBehavior() {
+    return { x: 0, y: 0 };
```

Comment:
According to documentation, this will also overwrite anchor navigation.
To be added as an improvement.

Reply:
Unclear what you mean by this. What has to be added?

Reply:
Should probably follow this piece of doc instead:

> Returning the savedPosition will result in a native-like behavior when navigating with back/forward buttons:
> ```
> scrollBehavior (to, from, savedPosition) {
>   if (savedPosition) {
>     return savedPosition
>   } else {
>     return { x: 0, y: 0 }
>   }
> }
> ```

Reply:
My point is that url#myLink won't work with those. We used to use them for search dojo results, or specific section post-save that you want to focus on.
Considering that's also a quite common (and expected) behavior, it's normally nice to have to avoid the "how the hell is this not working?" scenario

Reply:
Then I guess it should be this:
```javascript
scrollBehavior (to, from, savedPosition) {
  if (savedPosition) {
    return savedPosition;
  } else if (to.hash) {
    return {
      selector: to.hash,
    };
  } else {
    return { x: 0, y: 0 };
  }
}
```

Reply:
We can also specify an offset for when going the the hash, which will sort out our fixed header issue

Reply:
@campbs @grabartley Add an id to each section of dojo-detail page so we can hotlink the "upcoming event" section for exemple  and test this out with the offset :)